dali/operators/random/normal_distribution_gpu.cu

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <vector>
+#include <utility>
+#include "dali/core/convert.h"
+#include "dali/kernels/alloc.h"
+#include "dali/operators/random/rng_base_gpu.cuh"
+#include "dali/operators/random/normal_distribution.h"
+
+namespace dali {
+
+class NormalDistributionGPU : public NormalDistribution<GPUBackend, NormalDistributionGPU> {
+ public:
+  template <typename T>
+  struct Dist {
```

Comment:
Can this be moved to header, it seems pretty similar to the CPU variant.

Reply:
It can be moved to the header, but I don't expect any other file to need this class declaration so, should we? Operators are usually instantiated by name

Reply:
Just a suggestion. Up to you.

Reply:
It's not like we include operators' headers anywhere. That's hardly a public API.